### ISSUE TITLE
JAWS reads batch action bar which is invisible

### ISSUE THREAD

**Issue by jrpomeroy** [carbon-design-system/carbon#7200]
## Environment

> Operating system

Windows

> Browser

Latest Chrome

> Automated testing tool and ruleset

IBM CI 162 Checkpoint Violation: IBM 1.3.2 Meaningful Sequence, IBM 2.4.3 Focus Order
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/meaningful_sequence.html
https://www.ibm.com/able/guidelines/ci162/focus_order.html
Test Tool Type: JAWS Screen Reader 2019

> Assistive technology used to verify

## Detailed description

When JAWS reads the page content in order automatically (or using arrow keys), it will read the content of batch action bar ("Cancel"button, "Delete" button, etc.) before reading filter table, even the batch action bar is invisible.

In addition, if checkbox is checked and the batch action bar is displaying, user tries to use Up arrow to read the contents of batch action bar, JAWS will read the Filter table firstly which is incorrect.

> What version of the Carbon Design System are you using?

carbon-components-react@7.22.0

> What did you expect to happen?

Content is read in the correct order, and not at all when the batch actions are not displayed.

> What happened instead?

Batch actions read by JAWS when not displayed.

> What WCAG 2.1 checkpoint does the issue violate?

IBM CI 162 Checkpoint Violation: IBM 1.3.2 Meaningful Sequence, IBM 2.4.3 Focus Order

## Steps to reproduce the issue

Use JAWS to read page content. See description above.

> Please create a reduced test case in CodeSandbox

Use storybook here:
https://react.carbondesignsystem.com/?path=/story/datatable-batch-actions--usage

## Additional information

- Screenshots or code
- Notes

**Comment by mashenka123** [comment]
@tw15egan Could you please confirm when this issue can be triaged and fix? Thank you!

**Comment by xiepingp** [comment]
@tw15egan, is there any information for this issue?  We need the fix this month, could you please help kindly support? Thank you very much!

**Comment by xiepingp** [comment]
Hi @dakahn, I tested in https://react.carbondesignsystem.com/?path=/story/datatable-batch-actions--usage (v7.28.0), the invisible contents are still read by jaws:
1. jaws reads the content in order -> jaws reads the invisible batch action bar content 
![7200 - retest 1](https://user-images.githubusercontent.com/63960906/108308249-6b373c00-71ea-11eb-8bd2-62b457966dac.png)
2. tick the checkbox -> press Up arrow -> jaws reads the invisible Add button and Setting button:
![7200 - retest 2](https://user-images.githubusercontent.com/63960906/108308523-d719a480-71ea-11eb-9c22-ab950af23892.png)
Not sure if the version I tested includes the fix, could you please advise?

**Comment by joshblack** [comment]
Hi @xiepingp, this issue should be included in 10.29.0 which is available on `npm` but isn't tagged as `latest` yet. You can see the latest changes in this storybook if it helps to verify: https://carbon-components-react.netlify.com/

**Comment by xiepingp** [comment]
Hi @dakahn @joshblack, I retested in https://carbon-components-react.netlify.app/?path=/story/datatable-batch-actions--usage, the 1st part is ok now but the 2nd issue is still there:
1. jaws reads the content in order -> jaws reads the invisible batch action bar content -- **FIXED**
![7200 - fixed 1](https://user-images.githubusercontent.com/63960906/108800994-8382ce80-75cf-11eb-80f0-b9ac30b11a53.png)

2. tick the checkbox -> press Up arrow -> jaws reads the invisible Add button and Setting button -- **NOT FIXED**
![7200 - retest](https://user-images.githubusercontent.com/63960906/108801018-95647180-75cf-11eb-87d3-660c317ca457.png)

Please reopen the issue and could you have a look?

**Comment by mashenka123** [comment]
@joshblack Could you please reopen this?

**Comment by dakahn** [comment]
closed by https://github.com/carbon-design-system/carbon/pull/7789
